client/client.js

```diff
@@ -1,15 +1,14 @@
 $(function () {
 var socket = io();
 var INITIAL_USERNAME = "Bob";
-var AVATAR_SIZE = 15;
-$('#user').val(user);
+$('#user').val(INITIAL_USERNAME);
 $('form').submit(function(){
   socket.emit('chat message',{message: $('#m').val(), user: $('#user').val()});
```

Comment:
it might give you some value to pull some jQuery selectors into constants.

so f.x.
```
var INPUT_SELECTOR = "#m";
$(INPUT_SELECTOR).val('');
```

this will make it clearer what element exactly you're working with.

Reply:
you would then put `var INPUT_SELECTOR = "#m";` at the top of the file